src/Common/tests/System/PlatformDetection.cs

```diff
@@ -43,6 +43,15 @@ public static partial class PlatformDetection
 
         public static int WindowsVersion => GetWindowsVersion();
 
+        public static bool IsSmallerThan462
+        {
+            get
+            {
+                string description = RuntimeInformation.FrameworkDescription;
+                return IsFullFramework && !description.Contains("4.6.2") && !description.Contains("4.7");
```

Comment:
Oh, you pointed at it -- https://referencesource.microsoft.com/#System/parent/parent/parent/InternalApis/Clr/inc/AppContextDefaultValues.cs,69